packages/helpers/test/operations.test.js

```diff
@@ -36,3 +40,38 @@ describe('getOperationMessages integration test with AsyncAPI', () => {
   });
 });
 
+describe('getMessageExamples integration test with AsyncAPI', () => {
+  let operations;
+
+  beforeAll(async () => {
+    (operations = await getOperationsFromParsedDoc());
```

Comment:
all good just these paranthesis are pretty confusing, we do not need them, they do not change anything


```suggestion
    operations = await getOperationsFromParsedDoc();
```